### ISSUE TITLE
Crash using ssh-copy-id to unknown server

### ISSUE THREAD

**Issue by goerz** [blinksh/blink#140]
Using `ssh-copy-id` to a server that has not previously been connected to (such that the "authenticity can't be established") results in a crash. Immediately before crashing, the screen looks like this:
![img_0051](https://cloud.githubusercontent.com/assets/112306/18957658/340fc6fe-8660-11e6-9a74-f3299b93c639.PNG)

It works fine after logging in to the server once before using `ssh-copy-id`.

This is for the version that was released on the app store.

**Comment by carloscabanero** [comment]
Good catch. I couldn't replicate it with an "unknown / new" server. Could it a change on the fingerprint for that server?

**Comment by goerz** [comment]
No, I don't think it was a *change* of fingerprint. I noticed the problem immediately after installing the Blink version from the App store. I set up a host 'katamon' in the Hosts settings (one of the same hosts I was working on with the beta version), and then immediately tried to copy the default key to the host via `ssh-copy-id`. This would crash. Only by logging in once directly (manually entering my password, and confirming the identity of the host by typing 'yes'), I could then run `ssh-copy-id` afterwards without a crash.
